src/base/collection.js

```diff
@@ -749,18 +750,78 @@ CollectionBase.prototype._reset = function() {
 // Lodash methods that we want to implement on the Collection.
 // 90% of the core usefulness of Backbone Collections is actually implemented
 // right here:
-const methods = ['forEach', 'each', 'map', 'collect', 'reduce', 'foldl',
-  'inject', 'reduceRight', 'foldr', 'find', 'detect', 'filter', 'select',
-  'reject', 'every', 'all', 'some', 'any', 'include', 'contains', 'invoke',
-  'max', 'min', 'toArray', 'size', 'first', 'head', 'take', 'initial', 'rest',
-  'tail', 'drop', 'last', 'without', 'difference', 'indexOf', 'shuffle',
-  'lastIndexOf', 'isEmpty', 'chain'];
+
+const methods = {
```

Comment:
This adapter layer is for back compat right?

I think we should just do a breaking change and update all of these to use the newest names.

See [here](http://bookshelfjs.org/#Collection-subsection-lodash-methods) they all just ref the lodash docs (which will now be incorrect). I'd prefer we just updated the names in this `methods` array, and updated the docs [here](https://github.com/tgriesser/bookshelf/blob/master/src/base/collection.js#L589-L748).